src/components/Page.js

```diff
@@ -20,20 +19,17 @@ const Component = ({ children, classes }) => {
             <Style>{`
       body {background: #eee}
     `}</Style>
-            {/* <Grid alignItems="stretch" container justify="center" spacing={0}>
```

Comment:
You could potentially bring the grid back, but the inner grid will need to have `width: 100%`